diff --git a/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaProviderManager.kt b/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaProviderManager.kt
index de6f6854178236837e302cc06c244550170044ce..936e5defaf8467fa161918bc00a2fb73003ad442 100644
--- a/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaProviderManager.kt
+++ b/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaProviderManager.kt
@@ -5,15 +5,10 @@ import com.facebook.react.module.annotations.ReactModule
 import com.facebook.react.uimanager.ThemedReactContext
 import com.facebook.react.uimanager.UIManagerHelper
 import com.facebook.react.uimanager.ViewGroupManager
-import com.facebook.react.viewmanagers.RNCSafeAreaProviderManagerDelegate
-import com.facebook.react.viewmanagers.RNCSafeAreaProviderManagerInterface
-
 @ReactModule(name = SafeAreaProviderManager.REACT_CLASS)
-class SafeAreaProviderManager :
-    ViewGroupManager<SafeAreaProvider>(), RNCSafeAreaProviderManagerInterface<SafeAreaProvider> {
-  private val mDelegate = RNCSafeAreaProviderManagerDelegate(this)
+class SafeAreaProviderManager : ViewGroupManager<SafeAreaProvider>() {
 
-  override fun getDelegate() = mDelegate
+  override fun getDelegate() = null
 
   override fun getName() = REACT_CLASS
 
